enterprise/cloud.amazon/src/org/netbeans/modules/cloud/amazon/AmazonInstance.java

```diff
@@ -83,18 +85,25 @@
     
     public static final String DEFAULT_EMPTY_APPLICATION = "empty-nb-app-1.0.war";
     
+    private final String INSTANCEPROFILE_NAMESPACE = "aws:autoscaling:launchconfiguration"; // NOI18N
+    private final String INSTANCEPROFILE_OPTIONNAME = "IamInstanceProfile"; // NOI18N
+    private final String INSTANCEPROFILE_VALUE = "aws-elasticbeanstalk-ec2-role"; // NOI18N
+    
+    
```

Comment:
I would change this to a single line of separation.

Reply:
Done :)